doc/OVA/1.0.4/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -310,6 +310,8 @@ Restart CORTX
    
 Tested by:
 
+- June 20, 2021: Daniar Kurniawan (daniar@uchicago.edu) using OVA release 1.0.3 on MacOS Catalina running Oracle VirtualBox 5.2.18 (Qt5.6.3).
+
```

Comment:
Daniar, this wiki is for 1.0.4 as per newer OVA version published yesterday

Reply:
Yes, I am testing the 1.04 currently. I will update this PR soon.

Reply:
What is status of this PR please?

Reply:
It turns out that there is a problem with the newest version (1.0.4). I am stuck on `starting consul server`. I have been redoing all the steps and also trying to use rescue-OS and non rescue-OS, but so far the result is still the same. I am still investigating the problem. I will file an issue if the problem still exists.

Reply:
@daniarherikurniawan , do we still need this one?

Reply:
Hi @johnbent, Thanks for the reminder. I will close this for now. I haven't been able to run the newest OVA; I am still focusing on the S3 server.